### DISCUSSION TITLE
How to make `make` available inside of devshell?

### DISCUSSION THREAD

**Discussion by mainrs** [numtide/devshell#227] [Q&A] [answered]
I cannot figure out how to have `make` on my path when using devshell. It works for a normal `mkShell` configuration.

**Comment by firestack** [comment, marked as answer]
add [`gnumake`](https://search.nixos.org/packages?channel=22.05&show=gnumake&from=0&size=50&sort=relevance&type=packages&query=gnumake) to [`devshell.packages`](https://numtide.github.io/devshell/modules_schema.html#devshellpackages)

iirc `mkShell` includes autotools because `mkShell` is built around `nixpkgs.stdenv` which includes autotools by default.
